packages/@aws-cdk/aws-apigateway/lib/method.ts

```diff
@@ -54,9 +56,17 @@ export interface MethodOptions {
    */
   readonly requestParameters?: { [param: string]: boolean };
 
-  // TODO:
-  // - RequestValidatorId
-  // - RequestModels
+  /**
+   * The resources that are used for the response's content type. Specify request
+   * models as key-value pairs (string-to-string mapping), with a content type
+   * as the key and a Model resource name as the value
+   */
+  readonly requestModels?: { [param: string]: IModelRef };
+
+  /**
+   * The ID of the associated request validator.
+   */
+  readonly requestValidator?: IRequestValidatorRef;
```

Comment:
Why the `Ref` postfix?

Reply:
Removed. I wanted to avoid having to define all the other properties of IResource. I did (though I'm not really happy with the solution)